rq/job.py

```diff
@@ -514,8 +514,15 @@ def get_call_string(self):  # noqa
         if self.func_name is None:
             return None
 
-        arg_list = [repr(arg) for arg in self.args]
-        arg_list += ['%s=%r' % (k, v) for k, v in self.kwargs.items()]
+        # Python 2/3 compatibility
+        try:
+            arg_list = [repr(arg).decode('utf-8') for arg in self.args]
+        except AttributeError:
+            arg_list = [repr(arg) for arg in self.args]
```

Comment:
We already have a `as_text()` function that deals with the Py 2/3 differences, so you can just use that instead of doing a try/except here.